.github/CONTRIBUTING.md

```diff
@@ -28,7 +28,7 @@ It's also important to understand some overarching goals of Mocha, detailed belo
 
 ### :soccer: About Project Goals
 
-Mocha is a test framework.  Developers use it against anything from legacy spaghetti in IE7 to stage-0 TC39 features in Electron.  While still staying current, Mocha will only drop support for old platforms as a last resort.  If and only if Mocha cannot move forward as a project, support will be dropped.  If workarounds exist, they are preferred.
+Mocha is a test framework.  Developers use it against anything from legacy spaghetti in IE11 to stage-0 TC39 features in Electron.  Mocha is committed to providing support for maintained (LTS) versions of Node.js and popular browsers (of which IE11 is still one, as of December 2018).
```

Comment:
Be good to quantify what "popular browsers" means (if possible). e.g. >15% market share.

Reply:
I don't really keep up with this, so it's difficult to set expectations.  I've seen conflicting numbers for stuff as well.  when the articles start coming out about how we can all drop IE11 support, then we'll jump on the bandwagon--deal? 😝